app/Models/DocumentationPage.php

```diff
@@ -49,6 +49,11 @@ public function getRows(): array
                         'slug' => $slug,
                         'title' => $page->matter('title'),
                         'version_id' => $versionNumber,
+                        'github_link' => Str::of($package['path'])
```

Comment:
Instead of adding this in the model data, refactor it to `getGitHubLink()` method on the model, so that this isnt being called for every single page every single time the docs are being viewed

Reply:
Okay, and then call this method directly in the view ?

Reply:
yes

Reply:
hmm this is getting a bit difficult to get the url right without the file path. Can i add a relative path to the model data ?

Reply:
sure

Reply:
take a look again